packages/office-addin-project/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./lib/main.js",
   "scripts": {
     "build": "rimraf lib && concurrently \"tsc -p tsconfig.json\"",
-    "cli": "node lib/cli.js",
+    "cli": "node lib/cli.js convert -m test/invalid.manifest.xml",
```

Comment:
I don't think we want to package.json script command to have specific arguments like this.  Those should come from the command line (I believe that is how the others work).